charts/evm-rollup/values.yaml

```diff
@@ -183,7 +195,8 @@ config:
         condition: "{{ .Values.metrics.enabled }}"
       - name: txpool.nolocals
         value: "true"
-
+      - name: verbosity
```

Comment:
noting I believe other changes have been made to support verbosity on the evm chart since this was done, probably should remove this

Reply:
Done in [60a150b](https://github.com/astriaorg/astria/pull/2085/commits/60a150b91b5d990cf9d0e9a6b85b8a788c40cbac).